src/components/WebexSearchPeople/WebexSearchPeople.jsx

```diff
@@ -74,18 +75,23 @@ const WebexSearchPeople = forwardRef(({
       adapter.peopleAdapter.searchPeople(value)
         .subscribe(searchPeopleSuccess, onError);
     }
+
     if (typeof memberLookAhead === 'function') {
       setShowCollaborators(true);
       callMemberLookAhead(value);
     }
   };
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Even this eslint error suggests to implement useEffect instead of onChange handler directly being called

Reply:
This eslint error is only suggesting to add a dependency to the useCallback hook